deepcell_tracking/utils_test.py

```diff
@@ -161,6 +162,48 @@ def test_save_trks(self, tmpdir):
         np.testing.assert_array_equal(X, loaded['X'])
         np.testing.assert_array_equal(y, loaded['y'])
 
+        # test save trks to bytes
+        b = io.BytesIO()
+        utils.save_trks(b, lineage, X, y)
+
+        # load trks from bytes
+        b.seek(0)
+        loaded = utils.load_trks(b)
+        assert loaded['lineages'] == lineage
+        np.testing.assert_array_equal(X, loaded['X'])
+        np.testing.assert_array_equal(y, loaded['y'])
+
+    def test_save_trk(self, tmpdir):
+        X = get_image(30, 30)
+        y = np.random.randint(low=0, high=10, size=X.shape)
+        lineage = [dict()]
+
+        tempdir = str(tmpdir)
+        with pytest.raises(ValueError):
+            badfilename = os.path.join(tempdir, 'x.trks')
+            utils.save_trk(badfilename, lineage, X, y)
+
+        filename = os.path.join(tempdir, 'x.trk')
+        utils.save_trk(filename, lineage, X, y)
+        assert os.path.isfile(filename)
+
+        # test saved tracks can be loaded
+        loaded = utils.load_trks(filename)
+        assert loaded['lineages'] == lineage
+        np.testing.assert_array_equal(X, loaded['X'])
+        np.testing.assert_array_equal(y, loaded['y'])
+
+        # test save trks to bytes
+        b = io.BytesIO()
+        utils.save_trk(b, lineage, X, y)
+
+        # load trks from bytes
+        b.seek(0)
```

Comment:
Is this `seek` required? If so, should we just wrap that into `save_trk`?

Reply:
I was debating this, but I decided to match the convention that we will follow when saving npzs in the data-registry job code. If we leave `seek` out of `save_trk` then the only thing we need to do to support trks in this function is swap out `save_trk` for `np.savez_compressed`.

    def _upload(self, s3, bpath, X, y, lineages=None):
        # Save npz in memory
        bytes = BytesIO()
        np.savez_compressed(bytes, X=X, y=y)
        bytes.seek(0)


        # Upload to s3
        s3.upload_fileobj(bytes, self.in_bucket, bpath)